bundle/API/Search/Criterion/FullText.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\SiteBundle\API\Search\Criterion;
+
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\FullText as BaseFullText;
+use \Netgen\EzPlatformSearchExtra\API\Values\Content\Query\Criterion\FulltextSpellcheck;
+use Netgen\EzPlatformSearchExtra\API\Values\Content\SpellcheckQuery;
+
+class FullText extends BaseFullText implements FulltextSpellcheck
+{
+    /**
+     * Gets query to be used for spell check.
+     *
+     * @return \Netgen\EzPlatformSearchExtra\API\Values\Content\SpellcheckQuery
```

Comment:
No need for `@return` since you have a return type hint anyway.

Reply:
@emodric Fixed in https://github.com/netgen/site-bundle/pull/14/commits/d29931259d3c7e08abef9ea768050720e9e2182b.